lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2574,46 +2552,92 @@ void ModuleEmitter::emitStatementBlock(Block &body) {
 }
 
 //===----------------------------------------------------------------------===//
-// Module Driver
+// SplitModuleEmitter
 //===----------------------------------------------------------------------===//
 
+namespace {
+
+class SplitModuleEmitter {
+public:
+  explicit SplitModuleEmitter(StringRef dirname)
+      : encounteredError(false), dirname(dirname) {}
+
+  /// Whether any error has been encountered during emission.
+  std::atomic<bool> encounteredError;
```

Comment:
I learned the hard way that atomics need to be explicitly initialized with `= {}` for some reason.

Reply:
Ah the `= {}` is a good alternative to calling the `encounteredError(false)` constructor. Less verbose, more local. Forgot about the initializer lists.

Reply:
Oh, I didn't notice that.  Either way works for me.